docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - GAS_LIMIT=6700000
       - IPFS_HOST=ipfs
       - IPFS_PORT=5001
+      - IPNS_PATH=ipfs:5001/ipns/
```

Comment:
I would remove this locally and just build this wherever this is used like this:
```
IPFS_HOST = os.getenv("IPFS_HOST", "localhost")
IPFS_PORT = int(os.getenv("IPFS_PORT", 5001))
IPNS_PATH = os.getenv("IPNS_PATH", f"{IPFS_HOST}:{IPFS_PORT}/ipns/")
```
That way we can remove this from local `docker-compose.yml`